services/121-service/src/registration/dto/validate-registration-error-object.dto.ts

```diff
@@ -1,6 +1,6 @@
-export class ValidateRegistrationErrorObjectDto {
-  public lineNumber: number;
-  public column: string;
-  public value: string;
-  public error: string;
+export interface ValidateRegistrationErrorObject {
```

Comment:
> It looks like this interface is used as a return type for functions only. That means the file should be in the /interfaces folder, not the /dto folder. And the name of the interface should be in principle: FunctionNameReturnType. I think then: "ValidateAndCleanInputReturnType", as that seems the only public function. 

The public function `validateAndCleanInput` does not return the ValidateRegistrationErrorObject so I don't think this is appropriate in this case. This error object is only used in private function in the RegistrationsInputValidator class. Maybe it should live inside that file than instead of having a seperate interface file so it cannot be used in other files. Not sure what our practice is here. 

> All attributes of the interfaces should be defined readonly. This was in the guidelines only for class DTOs, but also goes for interfaces. I updated the guidelines.
(y)

Reply:
> The public function validateAndCleanInput does not return the ValidateRegistrationErrorObject so I don't think this is appropriate in this case.

You are right, I mislooked.

> This error object is only used in private functions in the RegistrationsInputValidator class.

This code smells, but refactoring is out of scope.

> What do you think?

I personally like to stick to "all interfaces in their own file". That makes it simple to follow.